CPRynner/CPRynner.py

```diff
@@ -3,130 +3,114 @@
 clusterview and runnoncluster plugins.
 """
 
+from future import *
 
 # Libsubmit creates a .script file in the working directory.
 # To avoid clutter, we run in a temp directory
-import tempfile
-import os
-
-import wx
-from rynner.core import Rynner
-from parsl.channels import SSHChannel
-from parsl.providers import SlurmProvider
-from parsl.launchers import SimpleLauncher
-from parsl.channels.errors import SSHException
-
+import tempfile, os
 workdir = tempfile.mkdtemp()
 os.chdir(workdir)
 
+import wx
+from Rynner.rynner.rynner import Rynner
+from libsubmit import SSHChannel
+from libsubmit.providers.slurm.slurm import SlurmProvider
+from libsubmit.launchers.launchers import SimpleLauncher
+from libsubmit.channels.errors import SSHException
+from cellprofiler_core.preferences import get_default_output_directory
 
-class ClusterSettingDialog(wx.Dialog):
+
+class clusterSettingDialog(wx.Dialog):
     """
     A dialog window for setting cluster parameters
     """
 
-    def __init__(self, cluster_address, tasks_per_node, work_dir, setup_script):
+    def __init__(self, cluster_address, tasks_per_node, work_dir, setup_script, run_command ):
         """Constructor"""
-        super().__init__(None, title="Login", size=(420, 480))
+        super(clusterSettingDialog, self).__init__(None, title="Login", size = (420,540))
 
         self.panel = wx.Panel(self)
 
         # cluster_address field
         cluster_address_sizer = wx.BoxSizer(wx.HORIZONTAL)
-        cluster_address_label = wx.StaticText(self.panel,
-                                              label="Cluster Address:",
-                                              size=(100, -1))
-        msg = (f"The URL of the cluster. If logged in, changing this will cause"
-               f" you to logout.")
-        cluster_address_label.SetToolTip(wx.ToolTip(msg))
-        cluster_address_sizer.Add(cluster_address_label, 0, wx.ALL | wx.CENTER,
-                                  5)
-        self.cluster_address = wx.TextCtrl(self.panel, value=cluster_address,
-                                           size=(300, -1), style=wx.TE_PROCESS_ENTER)
+        cluster_address_label = wx.StaticText(self.panel, label="Cluster Address:", size=(100, -1))
+        cluster_address_label.SetToolTip(wx.ToolTip(
+            "The URL of the cluster. If logged in, changing this will cause you to logout."
+        ))
+        cluster_address_sizer.Add(cluster_address_label, 0, wx.ALL|wx.CENTER, 5, )
+        self.cluster_address = wx.TextCtrl(self.panel, style = wx.TE_PROCESS_ENTER, value = cluster_address, size=(300, -1))
```

Comment:
A lot of this block and subsequent blocks are just reformatting code broken across multiple lines into single longer lines. In general it's best to avoid mixing this kind of large-scale reformatting into the same pull request as feature improvements or bug fixes, and also to avoid doing it without agreeing on a code style to be consistent to. What was the reasoning behind this change?